webrtc.html

```diff
@@ -3654,9 +3665,6 @@ <h2>Methods</h2>
       </section>
       <section>
         <h3>Legacy Interface Extensions</h3>
-          <div class="note">This section is broken out for readability. Consider
-          partial interfaces here to be part of their main counterparts, as they
-          overload existing methods.</div>
           <p>Supporting the methods in this section is optional. However,
           if these methods are supported it is mandatory to implement
           according to what is specified here.</p>
```

Comment:
Add a sentence saying that "due to WebIDL rules, the IDL syntax forthe legacy methods is given in <section>"?

Reply:
although I don't feel strongly about it, I don't think that's necessary - unless you know the spec as it is today by heart, you're unlikely to be surprised by having most of it in a single IDL fragment.

Reply:
The list of operations is what we usually list right under the IDL fragment. I'm looking to have the explanation of why these are listed here and not next to their IDL.